src/store/messages/saga.test.ts

```diff
@@ -32,6 +32,28 @@ describe('messages saga', () => {
       .run();
   });
 
+  it('send message', async () => {
+    const channelId = '0x000000000000000000000000000000000000000A';
+    const message = 'hello';
+    const mentionedUser = '00000000000000A';
```

Comment:
i don't think we need 'mentionedUser' here yet, as we don't have mentions at this point.

Reply:
We need `'mentionedUser' ` here for the test, because the `sendMessagesByChannelId` function used. 
` yield call(sendMessagesByChannelId, channelId, message, mentionedUser)`